modules/presets/secure-node.nix

```diff
@@ -132,8 +132,6 @@ in {
     services.onion-chef.enable = true;
     services.onion-chef.access.operator = [ "bitcoind" "clightning" "nginx" "liquidd" "spark-wallet" "electrs" "sshd" ];
 
-    # Unfortunately c-lightning doesn't allow setting the permissions of the rpc socket
-    # https://github.com/ElementsProject/lightning/issues/1366
     security.sudo.configFile =
      (optionalString cfg.clightning.enable ''
```

Comment:
Yeah I'm fine with keeping the lightning-cli option with sudo. It's simpler than having to deal groups.

Reply:
@erikarvstedt Re: 0e66ec47489732002ba9ab96dafd3ddd401a5ae8 I thought it would be good to keep, so `lightning-cli` would never be executed as root by accident. But I guess the simplicity win outweighs that. So ACK.